eng/Version.Details.xml

```diff
@@ -1,32 +1,32 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.3.24175.18" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.3.24165.20" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
```

Comment:
@jonathanpeppers you don't have preview4 here yet ?

Reply:
It was still broken this morning, close now:

* https://github.com/xamarin/xamarin-android/pull/8828